qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -5,7 +5,8 @@
 from typing import List, Dict, Optional
 
 import qcodes as qc
-from qcodes import VisaInstrument, DataSet
+from qcodes import VisaInstrument
+from qcodes.data.data_set import DataSet
```

Comment:
Internally this driver also calls qcodes.measure that should probably be fixed too

Reply:
Fixed at one place internally.